framework/src/main/java/org/checkerframework/framework/type/BoundsInitializer.java

```diff
@@ -279,32 +276,63 @@ private static void initializeExtendsBound(
      */
     private static class InitializerVisitor implements AnnotatedTypeVisitor<Void, Void> {
         /**
-         * The BoundStructure starting from the first wildcard or type variable bound initialization
-         * that kicked this visitation off.
+         * The RecursiveTypeStructure corresponding to the first wildcard or type variable bound
+         * initialization that kicked this visitation off.
          */
-        private final BoundStructure topLevelStructure;
+        private final RecursiveTypeStructure topLevelStructure;
 
-        private BoundStructure currentStructure = null;
+        /**
+         * The {@link RecursiveTypeStructure} corresponding to the wildcard or type variable that is
```

Comment:
When should an `@link` be used in javadoc and when not?
The documentation above for `topLevelStructure` doesn't link to the field type, whereas this does. Similar inconsistency for other javadoc below, where e.g. the key type is not linked, but the value type is.
Is it ever worth to `@link` these, as one can simply use the type from the signature?